crates/orderbook/openapi.yml

```diff
@@ -3,20 +3,20 @@ info:
   version: 0.0.1
   title: Order Book API
 servers:
-  - url: https://protocol-mainnet.dev.gnosisdev.com
-    description: Mainnet (Staging)
-  - url: https://protocol-mainnet.gnosis.io
-    description: Mainnet (Prod)
-  - url: https://protocol-rinkeby.dev.gnosisdev.com
-    description: Rinkeby (Staging)
-  - url: https://protocol-rinkeby.gnosis.io
-    description: Rinkeby (Prod)
-  - url: https://protocol-xdai.dev.gnosisdev.com
-    description: xDai (Staging)
-  - url: https://protocol-xdai.gnosis.io
-    description: xDai (Prod)
-  - url: http://localhost:8080
-    description: Local
+  - description: Mainnet (Staging)
+    url: https://barn.api.cow.fi/mainnet
+  - description: Mainnet (Prod)
+    url: https://api.cow.fi/mainnet
+  - description: Rinkeby (Staging)
+    url: https://barn.api.cow.fi/rinkeby
+  - description: Rinkeby (Prod)
+    url: https://api.cow.fi/rinkeby
+  - description: xDai (Staging)
+    url: https://barn.api.cow.fi/xdai
+  - description: xDai (Prod)
+    url: https://api.cow.fi/xdai
```

Comment:
Shouldn't those refer to Gnosis chain?

Reply:
Changed the description to mention `Gnosis Chain`. The URLs still use `xdai` as we haven't moved them over yet.

Reply:
Might be worth also updating the screen shot in the PR description.

Reply:
🦅 👀